test/distributed/cases/function/udf.sql

```diff
@@ -0,0 +1,22 @@
+-- @suite
+-- @setup
+
+-- @case
+-- @desc:test for create function (no arg)
+-- @label:bvt
+create function helloworld () returns int language sql as 'select id from test_table limit 1';
+
+-- @case
+-- @desc:test for create function (multiple args)
+-- @label:bvt
+create function twosum (x int, y int) returns int language sql as 'select $1 + $2' ;
```

Comment:
can we create function with same name but different number/type of params?   

test function name with mixed case, punctuation characters etc.

Reply:
Yes we can, sure I'll add this to test.